code/Entities/ClientEntity.php

```diff
@@ -163,6 +178,11 @@ public function getIdentifier()
         return $this->ClientIdentifier;
     }
 
+    public function setConfidential()
+    {
+        return $this->isConfidential = $this->ClientConfidential;
```

Comment:
`isConfidential` does not exists or used in the dependency files

Reply:
yeah think we are ok, we don't need to store both and to not refactor the ss module I'm leaving it as ClientConfidential in the DB. isConfidential is member in league/oauth-server